packages/core/tools/_mixins.scss

```diff
@@ -347,3 +307,27 @@
   height: 32px;
   width: 80px;
 }
+
+// Panel mixin
```

Comment:
The comments seem to have disappeared from the original panel mixin. How do I use it? May need to document in the breaking change log